.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+<!-- This template is for Devs to give QA details before moving the issue To-Test -->
+
+<!-- Link to the issue -->
+
+### Root cause
+<!-- Define findings related to the feature or bug issue. -->
+
+### What was fixed, or what changes have occurred
+<!-- Include information of the changes, including collateral areas which have been affected by this PR as requirement or for convenience. -->
+
```

Comment:
I also feel that we should add *technical notes summary* section so that it outlines technical steps that was done. This is to help the reviewer as well as any external parties or even QA if they can forsee things that might have impacted and we fail to mention in testing steps.

Reply:
I agree, we should distinguish between behaviors and technical changes, as may be more relevant for the reviewer.